### PR TITLE
Next release

### PR DESCRIPTION
**Pull request by ndelangen**
note: must add changelog

### CODE REVIEW

**Comment by tmeasday** [comment]
I'll do some QA today.

**Comment by tmeasday** [comment]
LGTM - I tested that:
 
 - I can still run builds correctly
 - I can still use the old options

I'm not quite sure how to test the action stuff but I think we can release this?

File: bin/main.js [hunk 1]
```diff
@@ -18,17 +18,7 @@ export async function parseArgv(argv) {
...
-      --do-not-start, -S  Don't attempt to start or build; use if your Storybook is already running
-      
-      --storybook-port <port>, -p  What port is your Storybook running on (auto detected from -s, if set)?'
-      --storybook-url <url>, -u  Storybook is already running at (external) url (implies -S)'
-      --storybook-build-dir, -d     <dirname>  Provide a directory with your built storybook; use if you've already built your storybook
-      --storybook-https  Use if Storybook is running on https (auto detected from -s, if set)?'
-      --storybook-cert <path>  Use if Storybook is running on https (auto detected from -s, if set)?'
-      --storybook-key <path>  Use if Storybook is running on https (auto detected from -s, if set)?'
-      --storybook-ca <ca>  Use if Storybook is running on https (auto detected from -s, if set)?'    
+      --storybook-build-dir, -d  <dirname>  Provide a directory with your built storybook; use if you've already built your storybook
```
Comment (domyen) [inline bin/main.js]:
```suggestion
      --storybook-build-dir, -d  <dirname>  Provide a directory with your built Storybook; use if you've already built your Storybook
```

File: bin/tester/index.js [hunk 2]
```diff
@@ -102,14 +105,26 @@ async function prepareAppOrBuild({
...
     }
 
+    const exists = await pathExists(path.join(buildDirName, 'iframe.html'));
+
+    if (!exists) {
+      if (buildScriptName) {
+        throw new Error(`Storybook was not build succesfully, there are likely errors above`);
+      } else {
+        throw new Error(dedent`
+          It looks like your storybook build (to directory: ${buildDirName}) failed, as that directory is empty. Perhaps something failed above?
```
Comment (domyen) [inline bin/tester/index.js]:
```suggestion
          It looks like your Storybook build (to directory: ${buildDirName}) failed, as that directory is empty. Perhaps something failed above?
```

File: bin/tester/runtimes.js [hunk 3]
```diff
@@ -47,22 +47,21 @@ export default async function getRuntimeSpecs(url, { verbose = false } = {}) {
...
-    (errors.length && !log.level.match(/silent/)) ||
-    (warnings.length && log.level.match(/verbose/))
-  ) {
+  const hasErrors = errors.length;
+  const hasVisibleErrors = hasErrors && log.level.match(/verbose/);
+  const hasVisibileWarnings = warnings.length && log.level.match(/verbose/);
+
+  if (hasVisibleErrors || hasVisibileWarnings) {
     log[errors.length ? 'error' : 'warn'](
       'The following problems were reported from your storybook:'
```
Comment (domyen) [inline bin/tester/runtimes.js]:
```suggestion
      'The following problems were reported from your Storybook:'
```

File: bin/tester/runtimes.js [hunk 4]
```diff
@@ -89,11 +88,20 @@ export default async function getRuntimeSpecs(url, { verbose = false } = {}) {
...
-        We suggest you fix the errors, but we will continue anyway..
-        ${separator}
-      `);
+    if (hasErrors) {
+      console.log(dedent`
+          This very likely caused some stories not working right or getting detected by Chromatic
+          Please fix the errors, we can't continue..
+          ${separator}
+        `);
+      throw new Error('Errors detected in storybook runtime');
```
Comment (domyen) [inline bin/tester/runtimes.js]:
```suggestion
      throw new Error('Errors detected in Storybook runtime');
```

**Comment by ndelangen** [comment]
blocked on https://github.com/chromaui/chromatic-cli/issues/75#issuecomment-577488895

**Pull request merged**